.github/workflows/geocoding-service-python.yml

```diff
@@ -1,17 +1,3 @@
-name: Geocoding location-service Python CI
-
-on:
-  push:
-    branches: [main, '*-stable']
-    paths:
-    - '.github/workflows/geocoding-service-python.yml'
-    - 'geocoding/location-service/**'
-  pull_request:
-    paths:
-    - '.github/workflows/geocoding-service-python.yml'
-    - 'geocoding/location-service/**'
-  workflow_dispatch:
-    paths:
```

Comment:
delete just this block and leave the above config intact?

Reply:
typo! thanks